src/DynamoCore/Core/AuthenticationManager.cs

```diff
@@ -11,6 +11,7 @@ namespace Dynamo.Core
     public class AuthenticationManager
     {
         private readonly IAuthProvider authProvider;
+        static LoginState? singleState = null;
```

Comment:
private?

Reply:
So if I understand correctly, this variable is for a one-time query of the login-in state whenever needed?

Reply:
yes | technically there is not much difference because both executes in the same flow only once time, but the mentioned approach looks more accurate and readable, updated.